.travis.yml

```diff
@@ -40,7 +49,7 @@ before_install:
 
 script:
     - ./build --configuration=$BUILD_CONFIGURATION
-    - ./build test+only --configuration=$BUILD_CONFIGURATION --where="cat!=FlakyNetwork"
+    - ./build test+only --configuration=$BUILD_CONFIGURATION --where="Category!=FlakyNetwork"
```

Comment:
I'm probably missing it, but I can't see anywhere else that this parameter name is changed?

Reply:
The NUnit test driver understands both cat and Category, but the .NET Core test driver only understands Category. I figured it was easier to use the common name instead of providing different strings. If the condition needs to get more complicated, though, you will probably need to have different strings.